site/design-notes/state-of-valhalla/03-vm-model.md

```diff
@@ -250,6 +253,12 @@ interface despite none of its supers doing so, the JVM has secretly made its
 mark on `C`.  Keeping secrets from reflection is a risky business, but in this
 case it seems safer to do so, lest a large fraction of existing Java classes
 suddenly grow new items in their `Class::getInterfaces` arrays.
+[ Hiding the injected interface from `Class::getInterfaces` seems to be one of
+those "solves today's problems while creating tomorrow's" and wouldn't pass
+muster if we were designing the feature without the weight of existing test
+suites. Are there other concerns than the existing test suites?  Given the
+other more significant changes in the ecosystem, I'm unclear why this one
+is worth avoiding.] 
```

Comment:
Agreement on today's call was to *not* hide the injected interfaces and use the preview phase to judge the fallout